CorpusSearch/Dockerfile

```diff
@@ -0,0 +1,33 @@
+﻿FROM mcr.microsoft.com/dotnet/aspnet:8.0 AS base
+RUN apt-get update && apt-get upgrade -y && \
+    apt-get install -y nodejs \
+    npm  
+USER $APP_UID
+WORKDIR /app
+EXPOSE 8080
+EXPOSE 8081
```

Comment:
Note: We're still set up to run on 5000/5001:

`      "applicationUrl": "https://localhost:5001;http://localhost:5000"`

I haven't kept up with the .net8 changes, could you drop a documentation link?

Reply:
Will change back to 5000,5001. Again autogenerated.